packages/itwin/grouping-mapping-widget/src/components/Mappings/MappingList.tsx

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type { Mapping } from "@itwin/insights-client";
+import React, { useEffect } from "react";
+import type { BeEvent } from "@itwin/core-bentley";
+import { MappingUIActionGroup } from "./MappingViewActionGroup";
+import { useState } from "react";
+import { useGroupingMappingApiConfig } from "../context/GroupingApiConfigContext";
+import { Anchor, ListItem, Text } from "@itwin/itwinui-react";
+import { ExtractionStates } from "./Extraction/ExtractionStatus";
+import { ExtractionStatus } from "./Extraction/ExtractionStatus";
+import { useFetchExtractionStates } from "./hooks/useFetchExtractionStates";
+import "./MappingList.scss";
+
+export interface MappingListProps {
+  selected: boolean;
+  onSelectionChange: (mapping: Mapping) => void;
+  mapping: Mapping;
+  jobStartEvent: BeEvent<(mappingId: string) => void>;
+  onClickMappingTitle?: (mapping: Mapping) => void;
+  onClickMappingModify?: (mapping: Mapping) => void;
+  onRefreshMappings: () => Promise<void>;
+  onToggleExtraction: (mapping: Mapping) => Promise<void>;
+  setShowDeleteModal: (mapping?: Mapping) => void;
+}
+
+export const MappingList = (props: MappingListProps) => {
```

Comment:
This is a misnomer. This is not a _list_, but rather a single a single _list item_.

Reply:
I have made this change.